### DISCUSSION TITLE
How to get textbox to lose focuse when clicked outside it in 11.0?

### DISCUSSION THREAD

**Discussion by lukaszossowski** [AvaloniaUI/Avalonia#12205] [Q&A] [answered]
In 11.0 preview-4 I just had my window focus on press:
```
  private void MainWindow_PointerPressed(object? sender, PointerPressedEventArgs e)
  {
      Focus();
  }
```

which removed focus from any textbox inside the window if one was focused. However after upgrading to latest stable release it no longer does that. The event still fires, but the textbox remains selected.

How do I get it to work like it originally did?

**Comment by maxkatz6** [comment, marked as answer]
Make sure window has Focusable=true, or another parent panel that you can focus. Background should also be non-null (Transparent works too).

**Reply by lukaszossowski** [reply]
Thanks, explicitly stating Focusable=true on the MainWindow fixed the issue.
